src/mono/mono/utils/lock-free-alloc.h

```diff
@@ -45,11 +45,7 @@ typedef struct {
 	MonoMemAccountType account_type;
 } MonoLockFreeAllocator;
 
-#ifdef HOST_WASM
-#define LOCK_FREE_ALLOC_SB_MAX_SIZE					(mono_opt_wasm_mmap ? 65536 : 16384)
-#else
 #define LOCK_FREE_ALLOC_SB_MAX_SIZE					16384
```

Comment:
we need a solution for the 16 vs 64k problem here since the system page size is still 64kb right now

Reply:
does `sbrk()` page size matter in any way ? Why ?

Reply:
i think it might be fine with the combination of all the changes you made, but mono_pagesize not agreeing with other sources of page size information feels like a hazard down the road

Reply:
what would be the other sources which could conflict ?